src/locale/locales/en.json

```diff
@@ -3,6 +3,10 @@
     "value": "Showing {0} rows and {1} columns.",
     "comment": "number of rows and columns announced by the screen reader. (tew 210929)"
   },
+  "SNTable.Accessibility.NavigationInstructions": {
+    "value": "Use arrow keys to navigate the cells and tab to move to pagination controls. See documentation for full instructions.",
```

Comment:
I am not sure that 'See documentation for full instructions' should be added here. The previous sentence is not good enough?

Reply:
Well, you could easily think that all you can do is navigate it, but there are lots of other controls

Reply:
I have checked this with casey, and it is fine.

Reply:
It'd probably be a better user experience if we could offer them a link to the topic in the help. Is that possible?

Reply:
Yeah, I thought of that too. Maybe... Since this is for screen readers, the users don't really have anything to click. I guess we could add a a shortcut to open a new tab with the instructions, then say `press x to open a tab with full instructions` or something. It is worth looking into, but we don't really have time now. I'll add a task for it and see if we want to add the link :)